### PR TITLE
Doc improvements

### PR DESCRIPTION
**Pull request by ian-r-rose**
Adds some supporting images about setting up your own client ID, and adds the start of a troubleshooting guide for users.

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Troubleshooting
+
+The `@jupyterlab/google-drive` extension is a complex plugin,
```
Comment (willingc) [inline docs/troubleshooting.md]: Perhaps:

```
...plug in. Adding to the complexity is the extension must...
```

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+in your browser to see if any errors are being shown.
+
+### The Google Drive panel doesn't show a login button.
+
+This means that the plugin may not be loading the Google API libraries,
+or they may not be initializing correctly.
+Check your internet connection and look at the Javascript console
+to see if any errors are being shown.
+
+### I am seeing an error saying `Not a valid origin for the client`.
```
Comment (willingc) [inline docs/troubleshooting.md]: Perhaps:  ~~I am seeing an error saying~~ `Not a valid origin for the client` error.

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+with the default one. This is configured to only work if you are running the
+Jupyter notebook server on `localhost`, ports `8888`-`8899`.
+If you are running on any other origins or ports, Google's servers will reject requests.
+If you cannot change your notebook server location, consider setting up your own client ID.
+
+If you have set up your own client ID for your JupyterLab deployment,
+then something is likely wrong with the configuration.
+Try looking through [advanced.md](./advanced.md) for a solution.
+
+### I am seeing an error that says `Failed to read the 'localStorage' property from 'Window'`.
```
Comment (willingc) [inline docs/troubleshooting.md]: ~~I am seeing an error that says~~ `Failed to read the 'localStorage' property from 'Window'` error

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+You may have your browser configured to block third-party cookies which is blocking Google Login.
+Either allow third-party cookies, or add an exception to the whitelist for `accounts.google.com`.
+
+### I have shared a document with another person, but they don't seem to by synchronizing!
+
+If two users are trying to collaborate on a document, but are using instances of JupyterLab
+configured with different client IDs, then synchronization WILL NOT WORK.
+Ensure that everybody is using the same client ID.
```
Comment (willingc) [inline docs/troubleshooting.md]: How do I tell what client ID that I am using?

**Comment by ian-r-rose** [comment]
Thanks, @willingc! Prepping follow-up.

**Comment by willingc** [comment]
Overall, the PR looks great. I really liked the screenshots.

**Pull request merged**